libs/pages/services/src/lib/ui/page-settings-deployment-pipeline/page-settings-deployment-pipeline.tsx

```diff
@@ -105,100 +156,69 @@ export function PageSettingsDeploymentPipeline(props: PageSettingsDeploymentPipe
           </Button>
         </SettingsHeading>
         <div className="h-full overflow-x-auto">
```

Comment:
`overflow-x-auto` isn't useful, but I think this `div` isn't necessary as well

Reply:
Fixed https://github.com/Qovery/console/pull/1671/commits/e912fe8aed53788c3a3457be1e6e503cf758b8fd